contracts/base/strategies/multi/pipes/Pipe.sol

```diff
@@ -14,12 +14,13 @@ pragma solidity 0.8.4;
 
 import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
 import "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
```

Comment:
import local libs

Reply:
To do that I've created IBVaultLocalOZ.sol with import of local open zeppelin lib. Otherwise we have to change PriceCalculator.sol and almost all infrastructure contracts.